game/src/devtools/fifteen_min/mod.rs

```diff
@@ -66,10 +66,24 @@ impl Viewer {
 }
 
 impl State<App> for Viewer {
-    fn event(&mut self, ctx: &mut EventCtx, _: &mut App) -> Transition {
+    fn event(&mut self, ctx: &mut EventCtx, app: &mut App) -> Transition {
         // Allow panning and zooming
         ctx.canvas_movement();
 
+        if ctx.redo_mouseover() {
+            app.recalculate_current_selection(ctx);
```

Comment:
`app.mouseover_unzoomed_buildings(ctx)`

There are a bunch of variations used in different places: https://github.com/dabreegster/abstreet/blob/29da713903404c8e2e7c1831c4d71e3a17daa732/game/src/app.rs#L265

Reply:
Thanks! I'm using that now. PTAL